multi/stm32l4-multi/libmulti/libuart.c

```diff
@@ -262,8 +262,10 @@ static int libuart_irqWrite(libuart_ctx *ctx, const void *buff, unsigned int buf
 
 	keepidle(1);
 
-	ctx->data.irq.txbeg = (void *)((unsigned char *)buff);
+	assert(ctx->data.irq.txbeg == NULL);
+
 	ctx->data.irq.txend = (void *)((unsigned char *)buff + bufflen);
+	ctx->data.irq.txbeg = (void *)((unsigned char *)buff);
```

Comment:
Note that in current irq implementation this should not matter as  `((ctx->base + cr1) & (1 << 7)) != 0` is never true outside code below.